### ISSUE TITLE
answerShippingQuery calls Error

### ISSUE THREAD

**Issue by dobeerman** [yagop/node-telegram-bot-api#448]
I'm trying to make an `answerShippingQuery` with code example below, but something went wrong. 
I've the following error:

> Unhandled rejection Error: ETELEGRAM: 400 Bad Request: shipping options are not specified

```
bot.on("shipping_query", shipping => {
  const ok = true;
  const opts = {
    shipping_options: [
      {
        id: "shipping",
        title: "Shipping by seller",
        prices: [
          { label: "Shipping by seller 1", amount: 500 },
          { label: "Shipping by seller 2", amount: 500 }
        ]
      },
      {
        id: "takeaway",
        title: "Take away",
        prices: [{ label: "-Take away", amount: 0 }]
      }
    ],
    error_message: ""
  };

  bot.answerShippingQuery(shipping.id, ok, opts);
});
```

**Comment by drGOD** [comment]
I have the same problem

**Comment by GochoMugo** [comment]
Fixed in **v0.30.0**.
